src/main/scala/com/fulcrumgenomics/util/Sequences.scala

```diff
@@ -101,6 +108,24 @@ object Sequences {
     count
   }
 
+  /**
+    * Returns the number of homopolymers of a given minimum length within the sequence.
+    *
+    * @param s a DNA or RNA sequence
+    * @param minLength the minimum length of homopolymer to count (must be greater than 0)
+    * @return the number of homopolymers of length >= minLength in the sequence
+    */
+  def homopolymers(s: String, minLength: Int): Int = {
+    require(minLength > 0, "minLength must be greater than 0")
+    var (homopolymers, currStreak) = (0, 1)
+    forloop(1, s.length) { basePosition =>
+      if (currStreak == minLength) homopolymers += 1
```

Comment:
this is cute, we count the homopolymer when it reaches `minLength`, not before, not after.